packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -1145,13 +1145,12 @@ const VirtualListBaseFactory = (type) => {
 		// render
 
 		getContainerClasses (className) {
-			let containerClass = null;
-
-			if (type === Native) {
-				containerClass = this.isPrimaryDirectionVertical ? css.vertical : css.horizontal;
-			}
-
-			return classNames(css.virtualList, containerClass, className);
+			return classNames(
+				css.virtualList,
+				this.isPrimaryDirectionVertical ? css.vertical : css.horizontal,
+				(type === Native) ? css.native : null,
```

Comment:
The `classNames` module accepts several convenient formats, including using key/value pairs to represent classname/truthyness, so this line could also be represented more easily as:
```
{[css.native]: (type === Native)}
```
Admittedly this looks a little complicated, but that's just because there's only one key/value pair in this object. Any number of keys can be in this object and it's the basis of how we structure adding styles in `kind()`. Also, when we accept one of a specific set of keywords as a prop, ex: `orientation: PropTypes.oneOf(['vertical', 'horizontal'])`, we can then easily dump that directly into the class list and use it as a class name:
```
	computed: {
		className: ({orientation, styler}) => styler.append(orientation),
	}
```
The propTypes limits the names allowed to one of our known class names, and we only have styles written for the known class names, so it all just works out.

Reply:
Good!